app/script/conversation/ConversationRepository.js

```diff
@@ -3491,6 +3491,20 @@ z.conversation.ConversationRepository = class ConversationRepository {
         if (conversationEntity && messageEntity) {
           const replacedEntity = conversationEntity.add_message(messageEntity, true);
           if (replacedEntity) {
+            const messages = conversationEntity.messages_unordered();
```

Comment:
Possible optimization, you can use the ordered messages list (namely `messages` I believe) and do stop when messages are before the timestamp of the edited message